### PR TITLE
Added a filter for ops with ref-valued inputs that are not supported.

### PR DESCRIPTION
**Pull request by eaplatanios**
Changes:
- Added compatibility with Python 3.
- Added a filter for ops with ref-valued inputs that are not supported.
- Regenerated bindings for TF commit 5e8df789cc30098d791475c14a623ec68b50b4ed with [PR #27327](https://github.com/tensorflow/tensorflow/pull/27327) on top of it.

The last one generates a large diff and not sure if it break compatibility with anything. Should I split it out as a separate PR?

### CODE REVIEW

File: generate_wrappers.py [hunk 1]
```diff
@@ -176,10 +177,11 @@ def __init__(self):
   def enum_codes(self):
     """Generates the swift code for enums."""
     codes = []
-    entries = list(self._entries.iteritems())
+    entries = list(six.iteritems(self._entries))
     for allowed_values, type_name in sorted(entries, key=lambda x: x[1]):
+      allowed_values = [str(a, encoding='utf-8') for a in allowed_values]
       codes.append(
-          # FIXME: Readd `@_frozen` after SR-9739 is resolved.
+          # FIXME: Read `@_frozen` after SR-9739 is resolved.
```
Comment (rxwei) [inline generate_wrappers.py]: “Readd” was correct.

**Review by rxwei — APPROVED** [review]
Thanks!

File: generate_wrappers.py (hunk 1 above)
Comment (eaplatanios) [reply generate_wrappers.py]: Sorry about that. I misinterpreted it while skimming through the file.

**Comment by rxwei** [comment]
> * Regenerated bindings for TF commit 5e8df789cc30098d791475c14a623ec68b50b4ed with [PR #27327](https://github.com/tensorflow/tensorflow/pull/27327) on top of it.

Since this contains some change that hasn't been merged yet, we should wait for it to merge first.

**Comment by eaplatanios** [comment]
@rxwei Sounds good! I removed the unmerged op. The generated module now contains everything from core TF repo commit 5e8df789cc30098d791475c14a623ec68b50b4ed.

**Comment by rxwei** [comment]
Great. Thanks.

File: RawOpsGenerated.swift [hunk 2]
```diff
@@ -2130,7 +1451,7 @@ public static func avgPool<T: FloatingPoint & TensorFlowScalar>(
   ksize: [Int32],
   strides: [Int32],
   padding: Padding,
-  dataFormat: DataFormat = .nhwc
+  dataFormat: DataFormat = .b'nhwc'
```
Comment (rxwei) [inline RawOpsGenerated.swift:1454]: @eaplatanios  This is malformed and causing https://github.com/apple/swift/pull/23700 to fail. I'm going to revert this PR for now.

Comment (eaplatanios) [reply RawOpsGenerated.swift:1454]: Oh I know why this happens. It’s a Python 3 compatibility bug. I’ll fix it once I get to my laptop.

Comment (eaplatanios) [reply RawOpsGenerated.swift:1454]: Fixed in #19 .

**Pull request merged**